crypto/fipsmodule/rand/entropy/entropy_source_tests.cc

```diff
@@ -0,0 +1,40 @@
+// Copyright Amazon.com Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0 OR ISC
+
+#include <gtest/gtest.h>
+
+#include "internal.h"
+
+#define MAX_MULTIPLE_FROM_RNG (16)
```

Comment:
What does the name mean? Is it number of iterations on calling the RNG?

Reply:
It's the bound `MAX_MULTIPLE_FROM_RNG * 8`.
Naming is hard.